### ISSUE TITLE
Quantize class requires being parsed in the first line of parse method

### ISSUE THREAD

**Issue by kapulkin** [jackwish/tflite2onnx#38]
**Bug description**

The bug is in first line with debug message if Quantize.parse method. Quantize.parse falls with assertion error.

```python

    @property
    def type(self):
        return 'QuantizeLinear' if self.isQuantize else 'DequantizeLinear' # requires being parsed indirectly

    @property
    def isQuantize(self):
        assert(self.status.parsed) # there is a requirement
        return self.inputs[0].dtype is TensorProto.FLOAT

    def parse(self):
        logger.debug("Parsing %s...", self.shorty) # shortly is a method in base class Operator, that reads type property
        op = self.tflite
        opcode = self.model.OperatorCodes(op.OpcodeIndex()).BuiltinCode()
        assert(opcode is tflite.BuiltinOperator.QUANTIZE or tflite.BuiltinOperator.DEQUANTIZE)

        assert(op.InputsLength() == 1)
        assert(op.OutputsLength() == 1)
        self.parseInput(0)
        self.parseOutput(0)

        self.setParsed()
```

Operator.shorty
```python

    @property
    def shorty(self):
        return '[%s](%s)' % (self.name, self.type)

```

**To Reproduce**

1. call tflite2onnx.convert on quantized model
2. debug log

```bash

[2020-12-26 21:28:26,239] DEBUG  tflite: data/u-net-model.tflite
[2020-12-26 21:28:26,239] DEBUG  onnx: data/u-net-resnet18-model.onnx
[2020-12-26 21:28:26,248] DEBUG  Parsing the Model...
[2020-12-26 21:28:26,248] DEBUG  Parsing the Graph...
[2020-12-26 21:28:26,248] DEBUG  Parsing operator: 0
<class 'tflite2onnx.op.quantize.Quantize'>
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/Users/stask/projects/motionLearning/chat/unet/broccole/convertTfliteToOnnx.py", line 37, in <module>
    main()
  File "/Users/stask/projects/motionLearning/chat/unet/broccole/convertTfliteToOnnx.py", line 34, in main
    tflite2onnx.convert(tfliteFilePath, onnxFilePath)
  File "/Users/stask/Library/Python/3.8/lib/python/site-packages/tflite2onnx/convert.py", line 44, in convert
    model.convert(explicit_layouts)
  File "/Users/stask/Library/Python/3.8/lib/python/site-packages/tflite2onnx/model.py", line 39, in convert
    self.parse()
  File "/Users/stask/Library/Python/3.8/lib/python/site-packages/tflite2onnx/model.py", line 31, in parse
    g.parse()
  File "/Users/stask/Library/Python/3.8/lib/python/site-packages/tflite2onnx/graph.py", line 63, in parse
    op.parse()
  File "/Users/stask/Library/Python/3.8/lib/python/site-packages/tflite2onnx/op/quantize.py", line 33, in parse
    logger.debug("Parsing %s...", self.shorty)
  File "/Users/stask/Library/Python/3.8/lib/python/site-packages/tflite2onnx/op/common.py", line 122, in shorty
    return '[%s](%s)' % (self.name, self.type)
  File "/Users/stask/Library/Python/3.8/lib/python/site-packages/tflite2onnx/op/quantize.py", line 25, in type
    return 'QuantizeLinear' if self.isQuantize else 'DequantizeLinear'
  File "/Users/stask/Library/Python/3.8/lib/python/site-packages/tflite2onnx/op/quantize.py", line 29, in isQuantize
    assert(self.status.parsed)
AssertionError

```

**Comment by jackwish** [comment]
@kapulkin This issue has been fixed in the latest `master` branch: https://github.com/jackwish/tflite2onnx/blob/master/tflite2onnx/op/quantize.py#L28

Please try with the latest `master` branch.

**Comment by jackwish** [comment]
Hi @kapulkin this issue should have been resolved by the recent `v0.3.1` release. Please let me know if there were an issue still.
